src/pages/user-profile.tsx

```diff
@@ -118,49 +141,81 @@ export const UserProfilePage = () => {
+                          className="object-cover h-full w-full transition-transform duration-300 -rotate-45 hover:rotate-0 hover:scale-110 scale-150"
+                        />
+                      </div>
+                      <div className="flex-1 min-w-0">
+                        <div className="flex justify-between items-start">
+                          <h2 className="text-md font-semibold text-gray-900 truncate">
+                            {snippet.unscoped_name}
+                          </h2>
+                          <div className="flex items-center gap-2">
+                            <TypeBadge
+                              type={snippet.snippet_type}
+                              className="mr-1"
+                            />
+                            <div className="flex items-center gap-1 text-gray-600">
+                              <StarIcon className="w-4 h-4 pt-[2.5px]" />
+                              <span className="text-[15.5px]">
+                                {snippet.star_count || 0}
+                              </span>
+                            </div>
+                            {isCurrentUserProfile && activeTab === "all" && (
+                              <DropdownMenu>
+                                <DropdownMenuTrigger asChild>
+                                  <Button
+                                    variant="ghost"
+                                    size="icon"
+                                    className="h-8 w-8"
+                                  >
+                                    <MoreVertical className="h-4 w-4" />
+                                  </Button>
+                                </DropdownMenuTrigger>
+                                <DropdownMenuContent>
+                                  <DropdownMenuItem
+                                    className="text-xs text-red-600"
+                                    onClick={(e) =>
+                                      handleDeleteClick(e, snippet)
+                                    }
+                                  >
+                                    <Trash2 className="mr-2 h-3 w-3" />
+                                    Delete Snippet
+                                  </DropdownMenuItem>
+                                </DropdownMenuContent>
+                              </DropdownMenu>
+                            )}
+                          </div>
                         </div>
-                        {snippet.is_private && (
-                          <div className="flex items-center text-gray-600">
-                            <LockClosedIcon className="w-4 h-4 mr-1" />
+                        <p className="text-sm text-gray-500 mb-[3px] truncate max-w-xs -mt-[1px]">
+                          {snippet.description
+                            ? snippet.description
+                            : "No description provided..."}
```

Comment:
```suggestion
                            : " "}
```

It should be empty and take up space